### ISSUE TITLE
Editorial (respec?) issue in the proof configuration section

### ISSUE THREAD

**Issue by iherman** [w3c/vc-di-ecdsa#67]
In [§3.2.5](https://www.w3.org/TR/vc-di-ecdsa/#proof-configuration-ecdsa-rdfc-2019) point (4) looks like that on my screen:

<img width="512" alt="Screenshot 2024-07-18 at 08 51 20" src="https://github.com/user-attachments/assets/c006b75a-be7c-4987-9ddc-452125a5f729">

it seems that the 

```html
           <li>
Set |proofConfig|.|@context| to
|unsecuredDocument|.|@context|.
            </li>
```

respec idiom does not work (I presume the `@` character is the source of the problems). Probably the safest is to use `<em>` or `<i>` explicitly, but that is really the editors' decision, I did not want to put this into a PR...

**Comment by TallTed** [comment]
> In [§3.2.5](https://www.w3.org/TR/vc-di-ecdsa/#proof-configuration-ecdsa-rdfc-2019)

Section numbers do not exist in the source. It is helpful to refer at least to the full section title (which I've also linked directly), i.e., [`Proof Configuration (ecdsa-rdfc-2019)`](https://github.com/w3c/vc-di-ecdsa/blob/03e285ce904abab4ecb141dd6c749c3102c3c9f7/index.html#L2794) and/or link to the specific source (it happens twice) —
* https://github.com/w3c/vc-di-ecdsa/blob/03e285ce904abab4ecb141dd6c749c3102c3c9f7/index.html#L835-L836
* https://github.com/w3c/vc-di-ecdsa/blob/03e285ce904abab4ecb141dd6c749c3102c3c9f7/index.html#L2830-L2831

> respec idiom does not work

Seems like an issue should be raised on respec, regardless of what workaround we use for our immediate needs.

**Comment by msporny** [comment]
Purely editorial, fixed in a4efc4457917f8077c5ec5c9ccc2e8d128db3322, closing.
